br/pkg/rtree/rtree.go

```diff
@@ -165,6 +208,31 @@ func (rangeTree *RangeTree) InsertRange(rg Range) *Range {
 	return out.(*Range)
 }
 
+func (rangeTree *RangeTree) Merge(splitSizeBytes, splitKeyCount uint64) []Range {
+	var mergeTargetIndex int = -1
+	sortedRanges := make([]Range, 0, rangeTree.Len())
+	rangeTree.Ascend(func(item btree.Item) bool {
+		if item == nil {
+			return false
+		}
```

Comment:
In what case this will happen? If this never happen, I think it is better to panic here. If it is possible, at lease we need to print some logs.

Reply:
Just keep the same as `Ascend` in `GetSortedRanges`